gateway/test/src/test/resources/test-plans/simple/simple-connector-interceptor-policy-testPlan.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testPlan xmlns="urn:io.apiman.test:2014:02:testPlan">
+
+  <testGroup name="Configure the Gateway">
+    <test name="Register Public Application">test-plan-data/simple/simple-connector-interceptor-policy/setup/001-publish-service.resttest</test>
+  </testGroup>
+
+  <testGroup name="Test the Connector Interceptor Policy through the Gateway">
+    <test name="Echo 1">test-plan-data/simple/simple-connector-interceptor-policy/connector-interceptor/001.resttest</test>
```

Comment:
Looks like you are invoking 001 but not 002-intercepted-response.  I think you need to add the latter into this list so that your feature actually gets tested!  :)